setup.sh

```diff
@@ -1,126 +1,223 @@
-/!/bin/bash
+#/!/bin/bash
 
+# A provisioning script for the created Vagrant vm.
+# This script is referred from the Vagrant file and it is executed during the
+# provisioning phase of starting a new vm.
+# Note: For this script to work there should be only one path in GOPATH env.
 # Everything in this file is run as root on the VM.
 
-set -e
-set -x
-
-echo "Setting up VM..."
-
 
-echo "Installing system tools..."
-yum -y install epel-release
-# Packages useful for testing/interacting with containers and
-# source control tools are so go get works properly.
-yum -y install yum-fastestmirror git mercurial subversion curl nc gcc etcd
+function installSystemTools() {
```

Comment:
can you change all function and variable names to underscore convention instead of camelcase? shell is usually underscores, as per https://google.github.io/styleguide/shell.xml#Naming_Conventions